lib/adapters/babel/6.x.coffee

```diff
@@ -43,6 +43,10 @@ class Babel extends Adapter
 
     data = { result: res.code }
     if res.map
+      # Convert source to absolute path.
+      # This is done for consistency with other accord adapters.
+      if res.map.sources then res.map.sources[0] = res.options.filename
```

Comment:
Does babel always return the primary file source as the first item in the array? Is it possible that there are multiple sources for a babel file?